src/components/ToDoContainer/index.js

```diff
@@ -1,11 +1,77 @@
-import React, { Component } from "react";
+import React, {
+  Component
+} from "react";
+
+import NewTodoForm from './NewTodoForm';
+import TodoList from './TodoList';
 
 class ToDoContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      newTodo: '',
+      todos: []
+    }
+
+    this.formSubmitted = this.formSubmitted.bind(this);
+    this.newTodoChanged = this.newTodoChanged.bind(this);
+    this.removeTodo = this.removeTodo.bind(this);
+    this.searchTodos = this.searchTodos.bind(this);
+  }
+
+  // Create new todo
+  newTodoChanged(event) {
```

Comment:
Define these methods using arrow notation - that way, you don't have to use the `.bind()` method!

`newTodoChanged = (event) => { ... }`